vast/vasttest/vasttest.go

```diff
@@ -81,15 +81,15 @@ func verifyElementName(t testing.TB, name string, xmlData []byte) {
 	}
 }
 
-// verifyNonEmptyXmlFields method verifies that all the fields defined in the model type has
+// verifyNonEmptyXMLFields method verifies that all the fields defined in the model type has
```

Comment:
verifyNonEmptyXMLFields method->verifyNonEmptyXMLFields function, same for others.

Reply:
Replace 'method with 'function' to make the description more accurate.